pkgs/by-name/ho/homer/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  fetchFromGitHub,
+  pnpm,
+  nodejs,
+  pkgs,
+}:
+pkgs.stdenv.mkDerivation rec {
+  pname = "homer";
+  version = "24.11.1";
+  src = fetchFromGitHub {
+    owner = "bastienwirtz";
+    repo = "homer";
+    rev = "v${version}";
+    hash = "sha256-a2t8qd/yxk7R0zyhQo0zoSIkfzfvBIrzNclNsD5GxRw=";
+  };
+
+  pnpmDeps = pnpm.fetchDeps {
+    inherit
+      pname
+      version
+      src
+      patches
+      ;
+    hash = "sha256-5unoY8lPaX9sZAJEBICpxSddwLV8liK1tbamB2ulvew=";
+  };
+
+  patches = [ ./sass-embedded.patch ];
```

Comment:
From skimming the patch I'm not really sure what it does, or why its needed. Maybe add a short comment explaining to make it easier for future maintainers.

Reply:
It fixes the lock file probably because it was made with an older version of pnpm

Reply:
would this be enough: "# Enables specifying a custom Sass compiler binary path via `SASS_EMBEDDED_BIN_PATH` environment variable."